src/components/tree/phyloTree/grid.js

```diff
@@ -147,7 +146,7 @@ const computeXGridPoints = (xmin, xmax, layout, distanceMeasure, minorTicks, pxA
   const minorGridPoints = [];
 
   /* step is the amount (same units of xmax, xmin) of seperation between major grid lines */
-  const [step, minorStep] = distanceMeasure === "num_date" ?
+  const {step, minorStep} = distanceMeasure === "num_date" ?
```

Comment:
The 2 functions which could be called here return objects with different property (names). I think this is the reason the grid doesn't appear for temporal axes but does for divergence.